backend/commons/src/test/java/org/sonarsource/sonarlint/core/commons/VersionTests.java

```diff
@@ -24,6 +24,12 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 class VersionTests {
+  @Test
+  void test_versioning_changes_handled_correctly() {
```

Comment:
Let's add another test also for Community Build versioning, just in case - it would be `25.1`